src/index.js

```diff
@@ -139,6 +141,12 @@ export default class Gantt {
         this.change_view_mode();
     }
 
+    change_popup_mode(mode = this.options.popup_trigger){
```

Comment:
I don't think we need to provide a method to change the popup event on the fly. It should only be set on initialization